bot/cogs/dm_comms.py

```diff
@@ -0,0 +1,48 @@
+from discord import DMChannel, Message
+from discord.ext import commands
+from rodhaj import Rodhaj
+
+
+class DmCommunications(commands.Cog):
+    def __init__(self, bot: Rodhaj):
+        self._bot = bot
+        self._pool = bot.pool
+
+    async def handle_user_no_thread(self, message: Message):
+        channel = message.channel
+        context = await self._bot.get_context(message)
+        result = await context.prompt(
+            "Seems like you do not have an active thread. "
+            + "Would you want to create one?",
+            timeout=600,
+            delete_after=True,
+        )
+
+        # Code to create thread here
+        if result:
+            await channel.send(
```

Comment:
`message.channel` sends it to the message's channel. I'd not recommend doing this as it can get confusing and does introduce a bug into the code. In reality, use the context here instead